pkg/webhooks/kafkatopic_validator.go

```diff
@@ -154,37 +161,61 @@ func (s *KafkaTopicValidator) checkKafka(ctx context.Context, topic *banzaicloud
 		return nil, errors.WrapIff(err, fmt.Sprintf("failed to list topics for kafka cluster: %s", topic.Spec.ClusterRef.Name))
 	}
 
+	var allErrs field.ErrorList
 	// The topic exists
 	if existing != nil {
 		// Check if this is the correct CR for this topic
 		topicCR := &banzaicloudv1alpha1.KafkaTopic{}
 		if err := s.Client.Get(ctx, types.NamespacedName{Name: topic.Name, Namespace: topic.Namespace}, topicCR); err != nil {
+			// Checking that the validation request is update
```

Comment:
```suggestion
			// Checking that the validation request is updated
```

Reply:
Since we're handling this in annotations everywhere else wouldn't it make more sense to add an annotation instead?
- We can add annotations for every newly created KafkaTopic with the kafkaTopic controller and for the existing KafkaTopic CR-s when there will be a reconcilation and there is no managedBy annotation. What if there is a missing managedBy annotation we every time force write the managedBy: koperator annotation?
What is the problem with to show that this topic certainly not managed by koperator as a confirmation in the status also. I would feel myself more relaxed when I delete a KafkaTopic Cr and I dont want to delete the topic itself when I can see in the status it is not managed by koperator.  This is only opinion of course.
Basicly the user set annotation to change the manager of the topic from koperator to something else, and he can get feedback from the status that this change has happened.

Reply:
It gives better user experience for me.

Reply:
I agree with Márton's reasoning here, and to me it makes sense to handle the "missing annotation" part with a status field that explicitly tells how it will be handled.